bin/replicate-mongodb.sh

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env bash
+
+set -eu
+
+if [[ "$#" == "0" ]]; then
+  echo "usage: $0 \$app"
+  exit 1
+fi
+
+app="$1"
+
+if echo "$app" | grep -q '_'; then
+  echo "app names do not have underscores in them"
+  exit 1
+fi
+
+replication_dir="${GOVUK_DOCKER_REPLICATION_DIR:-${GOVUK_DOCKER_DIR:-${GOVUK_ROOT_DIR:-$HOME/govuk}/govuk-docker}/replication}"
+
+bucket="govuk-integration-database-backups"
+
+case "$app" in
+  "authenticating-proxy")
+    hostname=router_backend
+    database=authenticating_proxy_production
+    ;;
+  "router"|"draft-router")
+    hostname=router_backend
+    database="${app//-/_}"
+    ;;
+  "asset-manager")
+    hostname=mongo
+    database=govuk_assets_production
+    ;;
+  "manuals-publisher"|"publisher"|"specialist-publisher")
+    hostname=mongo
+    database=govuk_content_production
+    ;;
+  *)
+    hostname=mongo
+    database="${app//-/_}_production"
+    ;;
+esac
+
+archive_dir="${replication_dir}/mongodb"
+archive_file="${hostname}.tar.gz"
+archive_path="${archive_dir}/${archive_file}"
+
+mkdir -p "$archive_dir"
+
+echo "Replicating mongodb for $app"
+
+if [[ -e "$archive_path" ]]; then
+  echo "Skipping download - remove ${archive_path} to force"
+else
+  remote_file_name=$(aws --profile govuk-integration s3 ls "s3://${bucket}/mongodb/daily/${hostname}/" | tail -n1 | sed 's/^.* .* .* //')
+  aws --profile govuk-integration s3 cp "s3://${bucket}/mongodb/daily/${hostname}/${remote_file_name}" "$archive_path"
+fi
+
+cat_cmd='cat'
+if command -v pv &>/dev/null; then
```

Comment:
Minor: since this is in the Brewfile, can we assume it'll be present?

Reply:
Ok, I'll remove this